frontend/pages/_app.js

```diff
@@ -1,6 +1,12 @@
+import { ToastProvider } from 'react-toast-notifications'
+
 import 'mobx-react-lite/batchingForReactDom'
 import '../styles.scss'
 
-const MyApp = ({ Component, pageProps }) => <Component {...pageProps} />
+const MyApp = ({ Component, pageProps }) => (
+  <ToastProvider autoDismiss autoDismissTimeout={3000} placement='bottom-right'>
```

Comment:
Em không nhớ là bottom right có cái nút Messenger à? Để sang bên trái hợp lý hơn

```suggestion
  <ToastProvider autoDismiss autoDismissTimeout={3000} placement='bottom-left'>
```